frontend/projects/upgrade/src/app/features/dashboard/home/components/experiment-design/experiment-design.component.ts

```diff
@@ -32,6 +33,8 @@ export class ExperimentDesignComponent implements OnInit, OnChanges, OnDestroy {
   @Input() currentContext: string;
   @Input() isContextChanged: boolean;
   @Input() animationCompleteStepperIndex: number;
+  @Input() dataChanged: false;
```

Comment:
you're setting `false` as the type, I think you want `@Input() dataChanged: boolean = false;`

Reply:
I see this in several files, let's change that for all of them

Reply:
resolved this